docs/installation.rst

```diff
@@ -144,52 +107,18 @@ To test if all the required files have been installed, please import STIPS in py
 
     >>> import stips
 
-The following warning message can be ignored if it appears::
-
-    WARNING: stips_data environment variable not found. Falling back on local STIPS data.
-
-
-Installing Using Docker
-#######################
+		    print(stips.__env__report__)
 
-Installing
-**********
-
-1. Start by installing the free `Docker Community Edition <https://www.docker.com/community-edition>`_ locally.
-This will make the `docker` command available in your terminal. Note that after installing docker,
-you must open the application once for docker to be available from the command line.
-
-2. You will need to clone the STIPS source code from the `spacetelescope/STScI-STIPS <https://github.com/spacetelescope/STScI-STIPS.git>`_ repository.
-`cd` into the directory you would like to store the source code and run::
-
-    git clone https://github.com/spacetelescope/STScI-STIPS.git
-
-    cd STScI-STIPS
+You should receive an output of the following form::
 
-3. Run the docker build command::
+		STIPS Version x.y.z with Data Version x.y.z at /Some/Path/To/stips_data
 
-    docker build -t stips .
+		STIPS Grid Generated with x.y.z
 
+		Pandeia version a.b.c with Data Version a.b.c. at /Some/Path/To/pandeia_refdata
 
+		Webbpsf Version d.e.f with Data Version d.e.f at /Some/Path/To/webbpsf_data_path
```

Comment:
The output block should also render as plain text instead of as Python code. Please replace this code snippet by copy/pasting the following over it:
```
You should receive an output of the following form:

.. code-block:: text

  STIPS Version x.y.z with Data Version x.y.z at /Some/Path/To/stips_data

  STIPS Grid Generated with x.y.z

  Pandeia version a.b.c with Data Version a.b.c. at /Some/Path/To/pandeia_refdata

  Webbpsf Version d.e.f with Data Version d.e.f at /Some/Path/To/webbpsf_data_path
```